### PR TITLE
Task/3659 scenarios section

### PR DESCRIPTION
**Pull request by glitch452**
image

### CODE REVIEW

**Review by maksvet — APPROVED** [review]

**Review by ReubenKogie — CHANGES_REQUESTED** [review]
- `Orbital Client -> Orbital Designer` to match update term?
- Just split the mockdefinition into two words. But feel free to ignore lol, as it's everywhere in the website
- Else, Looks great! 

Sorry, I erroneously committed the changes, wasn't sure how to do a draft commit but I figured it out eventually -_-

File: _pages/docs/scenarios/idnex.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (ReubenKogie) [inline _pages/docs/scenarios/idnex.md]: is the file name `_pages/docs/scenarios/index.md`? :-)

**Review by ReubenKogie — CHANGES_REQUESTED** [review]

File: _pages/docs/scenarios/idnex.md (hunk 1 above)
Comment (glitch452) [reply _pages/docs/scenarios/idnex.md]: lol yeah, good catch

**Comment by glitch452** [comment]
> * `Orbital Client -> Orbital Designer` to match update term?
> * Just split the mockdefinition into two words. But feel free to ignore lol, as it's everywhere in the website
> * Else, Looks great!
> 
> Sorry, I erroneously committed the changes, wasn't sure how to do a draft commit but I figured it out eventually -_-

Good call on Designer, I think I'll change mockdefinition back since that's how it's used all over, probably a good idea to try to be consistent.

**Review by ReubenKogie — APPROVED** [review]

**Pull request merged**